Integrations/DigitalDefense/DigitalDefense.py

```diff
@@ -0,0 +1,596 @@
+import demistomock as demisto
+from CommonServerPython import *
+''' IMPORTS '''
+import requests
+import json
+import sys
+import socket
+import struct
+from datetime import datetime, tzinfo, timedelta
+
+''' GLOBAL VARS '''
+API_TOKEN = demisto.params().get('apiToken')
+INCIDENT_VULN_SEVERITY = demisto.params().get('incidentSeverity')
+INCIDENT_FREQUENCY = demisto.params().get('incidentFrequency')
+FVM_URL = "https://vm.frontline.cloud"
+VULN_ENDPOINT = FVM_URL + "/api/scanresults/active/vulnerabilities/"
+HOST_ENDPOINT = FVM_URL + "/api/scanresults/active/hosts/"
+SCAN_ENDPOINT = FVM_URL + "/api/scans/"
+
+''' HELPER FUNCTIONS '''
+def get_fvm_data(request_url, api_token, **kwargs):
+    data = []
+    current_data = {}
+    resp = requests.get(request_url, headers={'Authorization': 'Token ' + str(API_TOKEN)}, timeout=30, **kwargs)
+    resp.raise_for_status()
+    current_data = json.loads(resp.text)
+    data.extend(current_data.get('results', []))
+    while current_data.get('next', None):
+        resp = requests.get(url=current_data.get('next'), headers={'Authorization': 'Token ' + str(API_TOKEN)}, timeout=30)
+        if not resp.ok:
+            demisto.debug("FrontlineVM get_fvm_data -- response error (status code: "+str(resp.status_code)+")")
+        current_data = {}
+        resp.raise_for_status()
+        current_data = json.loads(resp.text)
+        data.extend(current_data.get('results', []))
+    return data
+
+def parse_params(param_dict):
+    param_index = 0
+    new_param_dict = {}
+    for key in param_dict:
+        new_key = "_"+str(param_index)+"_"+key
+        new_param_dict[new_key] = param_dict[key]
+        param_index += 1
+    return new_param_dict
+
+
+def get_fetch_frequency_td():
+    ''' Returns the INCIDENT_FREQUENCY as a datetime.timedelta object    '''
+    fetch_frequency = INCIDENT_FREQUENCY.split()
+    demisto.debug("FrontlineVM get_fetch_incident_td -- using frequency: " + str(fetch_frequency))
+    if "min" in str(fetch_frequency[1]):
```

Comment:
INCIDENT_FREQUENCY is the user selected frequency at which to fetch incidents (in this case the users' vulnerabilities).

Since the frequency could be in the form of minutes or hours, we split the value and create a datetime object accordingly.